libnmstate/schema.py

```diff
@@ -165,11 +165,24 @@ class LinuxBridge(object):
     STP_HELLO_TIME = 'hello-time'
     STP_MAX_AGE = 'max-age'
 
+    VLAN_FILTERING_SUBTREE = 'vlan-filtering'
+    VLAN_FILTERING_ENABLED = 'enabled'
+
     PORT_SUBTREE = 'port'
     PORT_NAME = 'name'
     PORT_STP_PRIORITY = 'stp-priority'
     PORT_STP_HAIRPIN_MODE = 'stp-hairpin-mode'
     PORT_STP_PATH_COST = 'stp-path-cost'
+    PORT_VLAN_SUBTREE = 'vlan'
+    PORT_VLAN_TYPE = 'type'
+    PORT_VLAN_TRUNK_TAGS = 'trunk-tags'
+    PORT_VLAN_ACCESS_TAG = 'tag'
+    PORT_VLAN_ENABLE_NATIVE = 'enable-native'
+
+    PORT_VLAN_ENTRIES_ID = 'id'
+    PORT_VLAN_ENTRIES_ID_RANGE = 'id-range'
+    PORT_VLAN_ENTRIES_MIN_RANGE = 'min'
+    PORT_VLAN_ENTRIES_MAX_RANGE = 'max'
```

Comment:
It is unclear that this belongs below the IP_RANGE.
How about:
`LinuxBridge.Port.Vlan.TrunkTags.ID_RANGE_MAX`

We should start using sub-classes in general, it looks more readable.